src/ui/Windows/res/PasswordSafe3.rc2

```diff
@@ -963,6 +964,8 @@ BEGIN
   IDS_EDIT_NAMED_POLICY        "Change/View a Database Named Password Policy"
   IDS_TESTPOLICY               "Generate Password using selected Password Policy"
   IDS_CLICKTOCOPYGENPSWD       "Click to copy generated password to clipboard"
+  IDS_COPYPASSWORDHELP1        "Position 1 is the first character, 2 is the second etc. up to N for the last character of a password of length N. -1 is the last character, -2 the next to last etc. up to -N for the first password character. Positions must be separated by either spaces, commas or semi-colons to be recognised. There must also be a space, comma or semi-colon at the end of the list."
```

Comment:
The last sentence is surprising. Does the user really have to enter "1;3;5;" instead of "1;3;5"?